.github/styles/Vocab/Docs/accept.txt

```diff
@@ -48,6 +48,7 @@ httpie
 hyperloglogs
 hypertable
 Instana
+ip
```

Comment:
IP is acceptable already, so maybe try to replace `ip` with `IP` first. You'll not need to add here.

Reply:
in this case, make spell is blocking on the use of lowercase IP in my example (`example_ip`)

Reply:
Make sure that you use literal/code markup like you did in this comment, also in your text. The spell checker doesn't try to apply its stuff to words.

Reply:
yeah, I assumed that's what the double backticks would do, but Laysa showed me another way to achieve that result. Thanks for the heads up!